Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.19.5 AS deps
+FROM --platform=$BUILDPLATFORM golang:1.19.6 AS deps
```

Comment:
Seems like this change is coming up from the supporting arm64 build. I don't think we want this in this PR.